h/test/session_test.py

```diff
@@ -26,3 +27,21 @@ def test_sorts_groups(User):
 
     ids = [group['id'] for group in session_model['groups']]
     assert ids == ['__world__', 'c', 'a', 'b']
+
+
+@patch('h.session.features')
+def test_includes_features(features, fake_user):
+    feature_dict = {
+        'feature_one': True,
+        'feature_two': False,
+    }
+    features.all = Mock(return_value=feature_dict)
+    request = Mock(authenticated_user=fake_user)
```

Comment:
I'm still going to merge this but I will just add that I'm not sure the indirection of a fixture was really needed for this `fake_user` arg. You could just as easily have done `fake_user = Mock(groups=[])` on the preceding line.